api.py

```diff
@@ -1,54 +1,46 @@
 from fastapi import FastAPI
 import uvicorn
-import json
-import platform
-import os
+from pathlib import Path
```

Comment:
For now, we don't even need pathlib, because the data is at the root of the repository  
I used pathlib anyway in case we later want to put our data in a `data/` folder